gc/gc-base/src/main/java/org/projectnessie/gc/base/GCImpl.java

```diff
@@ -189,4 +200,12 @@ private static Instant getInstantFromMicros(Long microsSinceEpoch) {
         TimeUnit.MICROSECONDS.toNanos(
             Math.floorMod(microsSinceEpoch, TimeUnit.SECONDS.toMicros(1))));
   }
+
+  private static String getSerializedReference(ObjectMapper objectMapper, Reference reference) {
```

Comment:
this functionality should be defined exactly once and reused across classes/tests

Reply:
Moved to `GCUtil`